test/openal/test_openal_buffers.c

```diff
@@ -89,7 +89,9 @@ void iter() {
   alGetSourcef(source, AL_PITCH, &testPitch);
   assert(pitch == testPitch);
 #endif
-
+#ifdef TEST_FAST
```

Comment:
I wonder if this should be called `TEST_INTERACTIVE` and invert that check?

Reply:
Done.